tox_docker/__init__.py

```diff
@@ -307,7 +308,7 @@ def tox_runtest_pre(venv):  # noqa: C901
                 healthcheck=healthcheck,
                 labels={"tox_docker_container_name": container_name},
                 links=links,
-                name=container_name,
+                name=f"{container_name}_{random.randint(100000, 999999)}",
```

Comment:
How about using `os.getpid()` so that you can associate the container to the process that spawned it?

This would also be compatible with my suggestion about incorporating the `toxenv` into the container name.

```suggestion
                name=f"{container_name}_{toxenv}_{os.getpid()}",
```

Reply:
PID/toxenv/random number all approaches should solve the conflict, theoretically using PID might not work if we spin multiple containers using threads (thogh `tox -p` uses processes), so, using a simple random number should suffice.